solutions/fully-configurable/version.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">= 1.9.0"
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = ">= 2.16.1"
```

Comment:
DA should lock to a version (renovate will then bump it). Do not use range in DA (Todd posted this already above)